### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#60040]
#### VFS team name
Benefits Team 2

#### VFS product name
Financial Status Report (FSR)

#### VFS feature name
FSR Enhancements

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content doesn’t align with style guidance in VA.gov content style guide.

### Issue Details
Heading structure and focus management are inconsistent throughout the flow, in ways that could be confusing to screen reader users.

In some places, each page of the step has a unique H3 heading. In those steps, the H3 receives initial focus when arriving on the page and is the first thing a screen reader will announce. Sometimes that H3 is inside a `fieldset`, sometimes it's not.

In other places, there's a `legend` element for the `fieldset` on that page, and the `legend` is styled to look like an H3. But it's not actually an H3, and the "Step X of 7" H2 heading receives initial focus and is the first thing that's announced.

The net result is a somewhat jarring experience moving through the workflow. Sometimes you hear "Step X of 7" with the more general heading for the step. Then on another step you hear the more specific H3 for the contents of the page. Then on another step you hear "Step Z of 7" without a clear explanation of how you got from X to Z.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
Each page should follow a relatively consistent pattern. I think these two cases should cover it. (But please let me know if there's anything that doesn't fit!)

**Case 1:** The page only has one field on it. Then the page should be something like:

```
<h3>Unique heading describing this page</h3>

<label ...>Label for input</label>

<input ... />
```

**Case 2:** The page has multiple fields that make sense grouped together (eg. list of checkboxes). Then the page should be something like:

```
<fieldset>
   
   <legend>
      <h3>Unique heading describing this page</h3>
   </legend>

   <label ...>Label for first input</label>

   <input ... />

   <label ...>Label for next input</label>

   <input ... />

   [and so on]

</fieldset>
```

For screen reader users, each page has a unique H3 that then receives focus when landing on the page --- a consistent experience page-to-page.

For similar implementations, I'm pretty sure that Supplemental Claims did something along these lines, as well as some of what the Digitize Forms team has been working on. 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by HeatherRienks** [comment]
This work was completed in ticket #59289. Closing this ticket.

cc: @Tom-Davis
